lib/code_corps_web/controllers/stripe_connect_account_controller.ex

```diff
@@ -1,27 +1,47 @@
 defmodule CodeCorpsWeb.StripeConnectAccountController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
   alias CodeCorps.ConnUtils
   alias CodeCorps.StripeConnectAccount
   alias CodeCorps.StripeService.StripeConnectAccountService
+  alias CodeCorps.User
 
-  plug :load_and_authorize_changeset, model: StripeConnectAccount, only: [:create]
-  plug :load_and_authorize_resource, model: StripeConnectAccount, only: [:show, :update]
-  plug JaResource
+  action_fallback CodeCorpsWeb.FallbackController
+  plug CodeCorpsWeb.Plug.DataToAttributes
```

Comment:
don't forget `plug CodeCorpsWeb.IdsToIntegers` to

Reply:
I think I fixed all the issues now, thanks @begedin